lib/api/model/events.dart

```diff
@@ -265,6 +267,89 @@ class MessageEvent extends Event {
   }
 }
 
+/// A Zulip event of type `update_message`.
+@JsonSerializable(fieldRename: FieldRename.snake)
+class UpdateMessageEvent extends Event {
+  @override
+  String get type => 'update_message';
+
+  final int? userId; // TODO(server-5)
+  final bool? renderingOnly; // TODO(server-5)
+  final int messageId;
+  final List<int> messageIds;
+  final List<String> flags;
+  final int? editTimestamp; // TODO(server-5)
+  final String? streamName;
+  final int? streamId;
+  final int? newStreamId;
+  final String? propagateMode;
+  final String? origSubject;
+  final String? subject;
+  final List<Object>? topicLinks; // TODO shape of objects
```

Comment:
More precisely, the elements of this list will be `Map<String, dynamic>`. (More precisely than `Object`, that is.)

But if you don't feel like writing down the details, probably best to leave the whole field for later, like in `Message`:
```suggestion
  // final List<TopicLink> topicLinks; // TODO handle
```

In fact, given that we leave the field out in `Message`, probably best to leave it out here too and leave that work for a single follow-up task across `Message` and these events.